spinn_machine/machine.py

```diff
@@ -645,3 +645,25 @@ def total_cores(self):
         """
         return len([
             processor for chip in self.chips for processor in chip.processors])
+
+    def _has_wrap_arounds(self):
```

Comment:
why make a private method which is only called by the property? why not have the property do the calculation instead?